aom/src/main/java/com/nedap/archie/rminfo/ArchieModelNamingStrategy.java

```diff
@@ -27,12 +27,12 @@ public ArchieModelNamingStrategy(boolean standardCompliantExpressionNames) {
         this.standardsCompliantExpressionNames = standardCompliantExpressionNames;
     }
 
-    private static HashMap<String, String> rulesArchieToStandardTypeNamesMap = new HashMap<>();
-    {
+    private static final HashMap<String, String> rulesArchieToStandardTypeNamesMap = new HashMap<>();
+    static {
         rulesArchieToStandardTypeNamesMap.put("Operator", "EXPR_OPERATOR");
         rulesArchieToStandardTypeNamesMap.put("UnaryOperator", "EXPR_UNARY_OPERATOR");
         rulesArchieToStandardTypeNamesMap.put("BinaryOperator", "EXPR_BINARY_OPERATOR");
-        rulesArchieToStandardTypeNamesMap.put("Leaf", "EXPR_LITERAL");
+        rulesArchieToStandardTypeNamesMap.put("Leaf", "EXPR_LEAF");
         rulesArchieToStandardTypeNamesMap.put("Function", "EXPR_FUNCTION");
         rulesArchieToStandardTypeNamesMap.put("VariableReference", "EXPR_VARIABLE_REF");
         rulesArchieToStandardTypeNamesMap.put("Constant", "EXPR_CONSTANT");
```

Comment:
The line `rulesArchieToStandardTypeNamesMap.put("Constant", "EXPR_CONSTANT");` should be `  rulesArchieToStandardTypeNamesMap.put("Constant", "EXPR_LITERAL");`

Reply:
Done.